crypto/err/err.c

```diff
@@ -460,13 +451,11 @@ static unsigned long get_error_values(int inc, int top, const char **file,
     if (es->bottom == es->top)
         return 0;
 
-    if (top)
-        i = es->top;            /* last error */
-    else
-        i = (es->bottom + 1) % ERR_NUM_ERRORS; /* first error */
+    /* Which error, the top of stack (latest one) or the first one? */
+    i = g == EV_LAST ? es->top : (es->bottom + 1) % ERR_NUM_ERRORS;
```

Comment:
You have made this code less readable.

Reply:
Agrred. The  ?: operator is nice, but is sometimes not the best choice...